D/Darknet/build_tarballs.jl

```diff
@@ -2,34 +2,15 @@ using BinaryBuilder
 
 # Collection of sources required to build Darknet
 name = "Darknet"
-version = v"2019.11.15"
+version = v"2019.6.5"
 sources = [
-    "https://github.com/AlexeyAB/darknet/archive/71e835458904f782a905a06d28b4558d9e9830b4.zip" =>
-    "d77017462ae49f9ce2540c3e47589e68b2ce565573bf7d2f011b560bc989fbfa",
-    "./bundled",
+    "https://github.com/AlexeyAB/darknet/archive/3708b2e47d355ba0a206fd7a06bbc5a6e38af4ff.zip" =>
+    "e18a6374822fe3c9b95f2b6a4086decbdfbd1c589f2481ce5704a4384044ea6f",
```

Comment:
```suggestion
    ArchiveSource("https://github.com/AlexeyAB/darknet/archive/3708b2e47d355ba0a206fd7a06bbc5a6e38af4ff.zip",
                  "e18a6374822fe3c9b95f2b6a4086decbdfbd1c589f2481ce5704a4384044ea6f"),
```

Reply:
Caught this  from the depreciation warning 👍🏻